settings.gradle.kts

```
@@ -26,15 +26,19 @@ dependencyResolutionManagement {
 enableFeaturePreview("VERSION_CATALOGS")
 
 include(
-    ":appyx-components:backstack:android",
-    ":appyx-components:backstack:common",
-    ":appyx-components:demos:android",
-    ":appyx-components:demos:common",
-    ":appyx-components:demos:web",
-    ":appyx-components:internal:android",
-    ":appyx-components:internal:common",
-    ":appyx-components:spotlight:android",
-    ":appyx-components:spotlight:common",
+    ":appyx-components:stable:backstack:android",
+    ":appyx-components:stable:backstack:common",
+    ":appyx-components:internal:test-drive:android",
+    ":appyx-components:internal:test-drive:common",
+    ":appyx-components:stable:spotlight:android",
+    ":appyx-components:stable:spotlight:common",
```

Let's group the stable ones together.
```suggestion
    ":appyx-components:internal:test-drive:android",
    ":appyx-components:internal:test-drive:common",
    ":appyx-components:stable:backstack:android",
    ":appyx-components:stable:backstack:common",
    ":appyx-components:stable:spotlight:android",
    ":appyx-components:stable:spotlight:common",
```